drools-lsp-server/src/test/java/org/drools/lsp/server/DroolsLspDocumentServiceTest.java

```diff
@@ -14,7 +14,7 @@
 class DroolsLspDocumentServiceTest {
 
     @Test
-    void testCompletion() {
+    void completion() {
```

Comment:
Hi @tkobayas , thanks for the effort.
I noticed that you renamed another test to match the method actually tested, and I think it is good.
But here, `completion` is not really meaningfull. I would suggest to name the test as the method actually tested, eventually followed by "_" and something more specific, i.e. to differentiate tests that are targeting method with the same name: wdyt ?